domain/consensus/processes/ghostdagmanager/mergeset.go

```diff
@@ -82,3 +82,46 @@ func (gm *ghostdagManager) sortMergeSet(stagingArea *model.StagingArea, mergeSet
 	})
 	return err
 }
+
+func (gm *ghostdagManager) GetSortedMergeSet(stagingArea *model.StagingArea,
```

Comment:
Add comment that explains this function return the merge set sorted in a toplogical order